KomaMRIBase/src/datatypes/phantom/motion/simplemotion/Rotation.jl

```diff
@@ -2,49 +2,60 @@
 Rotation Motion
  
 Parameters:
-- Offset            (α) [1x1] 
-- Rotation axis     (n) [3x1]
-- Rotation center   (c) [3x1]
-- Angular velocity  (ω) [1x1]
+- Initial time   (ti)
+- Final time     (tf)
+- Rotation in x  (pitch)
+- Rotation in y  (roll)
+- Rotation in z  (yaw)
 
-ux = Un(ωt)(x - c) + (c - x)
+        [ 0,                      t <= ti
+pitch = [ pitch*(t-ti)/(tf-ti),   ti < t < tf
+        [ pitch,                  t >= tf
+
+        [ 0,                      t <= ti
+roll =  [ roll*(t-ti)/(tf-ti),    ti < t < tf
+        [ roll,                   t >= tf
+
+        [ 0,                      t <= ti
+yaw =   [ yaw*(t-ti)/(tf-ti),     ti < t < tf
+        [ yaw,                    t >= tf
+
+
+x' = cos(yaw)cos(roll)x + (cos(yaw)sin(roll)sin(pitch) - sin(yaw)cos(pitch))y + (cos(yaw)sin(roll)cos(pitch) + sin(yaw)sin(pitch))z
+y' = sin(yaw)cos(roll)x + (sin(yaw)sin(roll)sin(pitch) + cos(yaw)cos(pitch))y + (sin(yaw)sin(roll)cos(pitch) - cos(yaw)sin(pitch))z
+z' = -sin(roll)x        + cos(roll)sin(pitch)y                                + cos(roll)cos(pitch)z
```

Comment:
Use LateX, like in https://github.com/JuliaHealth/KomaMRI.jl/blob/master/KomaMRICore/src/datatypes/Spinor.jl. Also follow the docstring conventions used.